sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -439,7 +439,7 @@ public void VerifyImdsRequestFailureWithInvalidJsonPopulatesExceptionMessage()
         [NonParallelizable]
         [Test]
         [TestCase("""{"error":"invalid_request","error_description":"Identity not found"}""")]
-        [TestCase(null)]
+        //[TestCase(null)]
```

Comment:
TODO - open issue to uncomment this when Azure.Core releases with the new upgrade with System.Memory.Data. This should happen within a few days of this PR merging.

Reply:
I need to do this b/c the test only fails if System.Memory.Data version is 8.x. Azure.Identity gets the package transitively through Azure.Core so it doesn't currently have the new version.

I tried to hoist the version by referencing System.Memory.Data directly in Azure.Identity.csproj, but that causes a whole bunch of dependency conflicts because of all the shared dependencies between Azure.Core and System.Memory.Data.

These issues will all be resolved once I release Azure.Core with all the dependency upgrades I've done in the last few weeks, which I'm planning to do in the May release cycle. When I go to update the Azure.Core version in Packages.Data.Props after releasing, I will uncomment this test case.